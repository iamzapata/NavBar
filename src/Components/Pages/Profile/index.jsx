import React from "react"
import PageContainer from "Pages/PageContainer"

const Profile = () => <PageContainer>Hello Profile</PageContainer>

export default Profile
