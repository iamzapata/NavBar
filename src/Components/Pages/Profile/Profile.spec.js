import React from "react"
import { render, cleanup } from "react-testing-library"
import Profile from "./index"

describe("<Profile />", () => {
  afterEach(cleanup)
  it("Should render self", () => {
    const { container } = render(<Profile />)
    expect(container.firstChild).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
  })
})
