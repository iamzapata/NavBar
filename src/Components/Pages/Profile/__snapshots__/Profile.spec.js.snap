// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> Should render self 1`] = `
<div
  class="PageContainer"
>
  Hello Profile
</div>
`;
